webpack.config.jsdelivr.babel.js

```diff
@@ -16,6 +16,9 @@ export default {
         test: /\.js$/,
         exclude: /node_modules/,
         loader: 'babel-loader',
+        options: {
+          presets: [['@babel/preset-env', { modules: 'cjs' }]],
```

Comment:
Hey sorry if my question is a bit unaware but why is it `modules: cjs`? Is this not a browser library?

Reply:
It's indeed only used in the browser.

Reply:
Should we change the `cjs` @kakawajazz?